ops/terraform/services/eft/modules/bfd_eft_outbound_o11y/lambda_src/outbound_slack_notifier/requirements.in

```diff
@@ -1,2 +1,3 @@
 aws-lambda-powertools[all]
 setuptools>=69.0.2
+
```

Comment:
```suggestion
```

Reply:
file modified removed /r/n from end-of-file